db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java

```diff
@@ -320,8 +320,8 @@ public void printSubNodes(int depth){
     public FromList getFromList(){ return fromList; }
 
     /**
-     * Remap the column reference node to underlying column reference in the select, if possible.
-     * The select node here corresponds to a union branch
+     * Given a column reference, find the corresponding column reference in the select, if possible.
```

Comment:
I'd word it as follows:

```
Finds the column reference (as used in select) for a given column reference
```

The rest is in `@param` and `@return`

Reply:
Thanks. I've updated the comment accordingly.